simplenote_cli/view_note.py

```diff
@@ -30,13 +30,13 @@ def get_note_content_as_list(self):
         if not self.key:
             return lines
         if self.old_note:
-            for l in self.old_note['content'].split('\n'):
+            for l in self.old_note['content'].splitlines():
                 lines.append(
                     urwid.AttrMap(urwid.Text(l.replace('\t', ' ' * self.tabstop)),
                                   'note_content_old',
                                   'note_content_old_focus'))
         else:
-            for l in self.note['content'].split('\n'):
+            for l in self.note['content'].splitlines():
```

Comment:
I'm a little bother by splitlines; I've heard of some strange/unexpected behaviour.  For example [this stackoverflow question](https://stackoverflow.com/questions/24453713/python-string-splitlines-removes-certain-unicode-control-characters).

Perhaps it would be safer to use a regex as described in the answer to the SO post linked above?